src/store/reducers.js

```diff
@@ -0,0 +1,55 @@
+const defaultState = {
+  trendingGifs: [],
+  favoriteGifs: [],
+  error: '',
+  loading: false,
+  singleGif: null
+}
+
+const reducers = (state = defaultState, action) => {
+  switch(action.type) {
+    case 'LOADING_START':
```

Comment:
Also, it is important to define the action types as constants and defined their respective action creator.